library/Rules/MinimumAge.php

```diff
@@ -20,16 +21,16 @@ class MinimumAge extends AbstractRule
 
     public function __construct($age, $format = null)
     {
+        if (!filter_var($age, FILTER_VALIDATE_INT)) {
+            throw new ComponentException('The age must be a numeric value.');
+        }
+
         $this->age = $age;
         $this->format = $format;
     }
 
     public function validate($input)
     {
-        if (!is_int($this->age)) {
```

Comment:
But, you have to send another pull request against 0.8 branch to fix that on those versions.